relay-general/src/store/normalize/span/description.rs

```diff
@@ -2,15 +2,59 @@
 use std::borrow::Cow;
 use std::collections::BTreeMap;
 
+use once_cell::sync::Lazy;
+use regex::Regex;
+
 use crate::protocol::Span;
-use crate::store::regexes::{
-    REDIS_COMMAND_REGEX, RESOURCE_NORMALIZER_REGEX, SQL_ALREADY_NORMALIZED_REGEX,
-    SQL_NORMALIZER_REGEX,
-};
+use crate::store::regexes::{REDIS_COMMAND_REGEX, RESOURCE_NORMALIZER_REGEX};
 use crate::store::{scrub_identifiers, scrub_identifiers_with_regex, SpanDescriptionRule};
 use crate::types::{Annotated, ProcessingAction, ProcessingResult, Remark, RemarkType, Value};
 
-// TODO: docs
+/// Regex with multiple capture groups for SQL tokens we should scrub.
+///
+/// Slightly modified from
+/// <https://github.com/getsentry/sentry/blob/65fb6fdaa0080b824ab71559ce025a9ec6818b3e/src/sentry/spans/grouping/strategy/base.py#L170>
+/// <https://github.com/getsentry/sentry/blob/17af7efe869007f85c5322e48aa9f80a8515bde4/src/sentry/spans/grouping/strategy/base.py#L163>
+static SQL_NORMALIZER_REGEX: Lazy<Regex> = Lazy::new(|| {
+    Regex::new(
+        r#"(?xi)
+        # Capture `SAVEPOINT` savepoints.
+        ((?-x)SAVEPOINT (?P<savepoint>(?:(?:"[^"]+")|(?:'[^']+')|(?:`[^`]+`)|(?:[a-z]\w+)))) |
+        # Capture single-quoted strings, including the remaining substring if `\'` is found.
+        ((?-x)(?P<single_quoted_strs>'(?:\\'|[^'])*(?:'|$))) |
+        # Capture placeholders.
+        ((?-x)(?P<placeholder>(?:\?+|\$\d+))) |
+        # Capture numbers.
+        ((?-x)(?P<number>(-?\b(?:[0-9]+\.)?[0-9]+(?:[eE][+-]?[0-9]+)?\b))) |
+        # Capture booleans (as full tokens, not as substrings of other tokens).
+        ((?-x)(?P<bool>(\b(?:true|false)\b)))
+        "#,
+    )
+    .unwrap()
+});
+
+/// Regex to make multiple placeholders collapse into one.
+/// This can be used as a second pass after [`SQL_NORMALIZER_REGEX`].
+static SQL_COLLAPSE_REGEX: Lazy<Regex> = Lazy::new(|| {
+    Regex::new(
+        r#"(?xi)
+        (VALUES|IN) \s+ \( (?P<values> ( %s ( \)\s*,\s*\(\s*%s | \s*,\s*%s )* )) \)?
```

Comment:
Should we prefix this line with `(?-x)`? The `x` above turns the free spacing mode on.